src/features/DisplayControls/NursingTasks/components/UpdateNursingTasks.jsx

```diff
@@ -63,35 +120,85 @@ const UpdateNursingTasks = (props) => {
     Object.keys(tasks).forEach((key) => {
       if (tasks[key].isSelected) {
         saveDisabled = false;
+        setIsAnyMedicationAdministered(true);
       }
     });
     updateIsSaveDisabled(saveDisabled);
   };
 
-  const handleToggle = (checked, id) => {
-    updateTasks({
-      ...tasks,
-      [id]: {
-        ...tasks[id],
-        isSelected: checked,
-        actualTime: checked ? moment() : null,
-      },
-    });
+  const timeToEpoch = (time) => {
+    const [hours, minutes] = time.split(":").map(Number);
+    const specificTime = new Date();
+    specificTime.setHours(hours, minutes, 0, 0);
+    const epochTimeInSeconds = Math.floor(specificTime.getTime() / 1000);
+    return epochTimeInSeconds;
```

Comment:
it seems like a generic utility method, can we move it to common utility so it could be used at other places too in future

Reply:
done